index.js

```diff
@@ -416,6 +431,32 @@ Primus.readable('forEach', function forEach(fn, done) {
 });
 
 /**
+ * Send a ping packet to all clients to ensure that they are still connected.
+ *
+ * @returns {Primus}
+ * @api private
+ */
+Primus.readable('heartbeat', function heartbeat() {
+  this.forEach(function forEach(spark) {
+    if (!spark.alive) {
```

Comment:
An official mechanism for disabling this per-spark would be very helpful. I have some clients who use Primus and others who do not, and those who do not should not receive these. Perhaps this could call a function on the Spark itself? I am happy to contribute a patch.